src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -46,20 +45,29 @@
 from dsp_tools.utils.logger_config import logger_savepath
 
 
+@dataclass(frozen=True)
```

Comment:
I guess this should live in the dedicated `src/dsp_tools/commands/xmlupload/models` package, where all other models live